plugins/toolbox/src/components/DefaultEditor/JsonSpaceSelector.tsx

```diff
@@ -25,16 +27,16 @@ export const JsonSpaceSelector = (props: {
       }}
     >
       <MenuItem value={2} sx={{ p: '6px 16px !important' }}>
-        2 spaces
+        {t('components.jsonSpaceSelector.count', { spaceCount: '2' })}
```

Comment:
You can actually use the automatic pluralization feature of i18next here. The variable must be named `count` and you need three versions. See i18next docs and https://backstage.io/docs/plugins/internationalization/#plurals

Reply:
done